lightningd/options.c

```diff
@@ -434,13 +433,14 @@ static void dev_register_opts(struct lightningd *ld)
 }
 #endif
 
-static const struct config testnet_config = {
+
+static const struct config groestlcoin_testnet_config = {
```

Comment:
Since we are plain c-grs-lightning, we should move back using to testnet_config to make it easier to merge with upstream